.travis.yml

```diff
@@ -2,23 +2,13 @@ language: python
 cache: pip
 
 python:
-  - "3.5"
-  - "2.7"
+  - "3.6"
+  - "3.7"
 
 env:
-  - DJANGO="django>=1.11,<1.12"
-  - DJANGO="django>=2.0,<2.1"
   - DJANGO="django>=2.1,<2.2"
   - DJANGO="django>=2.2,<2.3"
-
-matrix:
-  exclude:
-   - python: "2.7"
-     env: DJANGO="django>=2.0,<2.1"
-   - python: "2.7"
-     env: DJANGO="django>=2.1,<2.2"
-   - python: "2.7"
-     env: DJANGO="django>=2.2,<2.3"
+  - DJANGO="django>=3.0,<3.2"
```

Comment:
"<3.2" ? Did you just make this change or what this before as well?
This should clearly be 3.1, why is this 3.2?

Reply:
Typo :sweat_smile: